src/Form/MetadataDisplayForm.php

```diff
@@ -109,7 +109,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['preview']['render_native'] = [
       '#type' => 'checkbox',
       '#defaut_value' => FALSE,
-      '#title' => 'Show Preview using native Output Format (e.g HTML)',
+      '#title' => 'Show Preview using native Output Format (e.g HTML) and output errors if there are any.',
       '#states' => [
         'visible' => ['input[name="ado_context_preview"' => ['filled' => true]],
```

Comment:
I think this will revert the fix you already did to #states? Hope note. Please check

Reply:
Whoops, forgot to rebase. Just fixed now.